src/lib/json-path-errors.js

```diff
@@ -15,6 +15,12 @@ module.exports = (definition) => JSONPath({ json: definition, path: '$..[InputPa
       JSONPath({ path, json: definition });
       return null;
     } catch (e) {
+      try {
+        JSONPath({ path, json: [{}] });
+        return null;
+      } catch (e2) {
+        // ignore, return first error
+      }
```

Comment:
workaround for now

will have a better fix for this with the asl-path-validator. I see now that I need to update the grammar to support the atmark outside of a filter.